iwyu.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+###
+# Checks source with include-what-you-use by running cmake in a temporary build
+# folder to obtain a compile database.
+###
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+iwyu_ver=$(include-what-you-use --version | awk '{print $2}')
+iwyu_ver_maj=$(echo "$iwyu_ver" | cut -d. -f 1)
+iwyu_ver_min=$(echo "$iwyu_ver" | cut -d. -f 2)
+
+if ! (( iwyu_ver_maj > 0 || iwyu_ver_min >= 11 )); then
+    echo >&2 "Error: Expecting include-what-you-use version >= 0.11, got $iwyu_ver"
+    exit 1
+fi
+
+tmpdir=$(mktemp -d)
+trap "{ rm -rf $tmpdir; }" EXIT
```

Comment:
Do we really need this? The combination of `rm -rf` together with an unquoted variable looks **very fragile**...

Reply:
Strictly speaking it's not necessary, but useful for running the script manually, outside of a throwaway Docker container. In bash, it's not as easy to do as in other languages, but I'd rather not leave temporary files behind.

You're right about the quoting though, I've fixed it.